config/crd/bases/config.kubeflow.org_kfpcontrollerconfigs.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     singular: kfpcontrollerconfig
   scope: Namespaced
   versions:
-  - name: v2
```

Comment:
Why was this v2?!

Reply:
It was autogenerated by the CLI. Note that this is the spec for the controller config.
this here gives differing suggestions for what it should be: https://book.kubebuilder.io/component-config-tutorial/define-custom-config.html
IMO this should be v1alpha1 as well